client/components/Auth.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+
+
```

Comment:
This `<Auth />` component is very important. It won't render any of it's children unless the user is logged in. Generally, a Higher Order Component is used for this case, but with react router, HOC's do not work the same way. The more "reactive" way to achieve this is to create a protected parent route, as I've done here